content/docs/lifting-state-up.md

```diff
@@ -186,11 +186,11 @@ First, we will replace `this.state.temperature` with `this.props.temperature` in
     // ...
 ```
 
-We know that [props are read-only](/docs/components-and-props.html#props-are-read-only). When the `temperature` was in the local state, the `TemperatureInput` could just call `this.setState()` to change it. However, now that the `temperature` is coming from the parent as a prop, the `TemperatureInput` has no control over it.
+हम जानते हैं कि [props केवल-पढ़ने के लिए हैं](/docs/components-and-props.html#props-are-read-only). जब `temperature` लोकल state में है, तो `TemperatureInput` इसे बदलने के लिए `this.setState()` को कॉल कर सकता है। हालाँकि, अब `temperature` अपने पैरेंट से prop बनकर आ रहा है, `TemperatureInput' का इस पर कोई नियंत्रण नहीं है।
```

Comment:
केवल-पढ़ने के लिए -> does not mean read only

Reply:
जब `temperature` लोकल state में tha

Reply:
जब `temperature` लोकल state में है, तो `TemperatureInput` इसे बदलने के लिए `this.setState()` को कॉल कर सकता है। -> revisit

Reply:
next line too